src/main/java/seedu/mtracker/commands/FindCommand.java

```diff
@@ -8,25 +8,25 @@
 public class FindCommand extends Command {
     public static final String COMMAND_WORD = "find";
     public static final String EMPTY_STR = "";
-    private String keyword;
+    private String searchString;
     private ArrayList<Instrument> filteredInstruments;
```

Comment:
If i am not wrong java ide says this can be converted to local variable as its only used in execute, so to reduce the scope lets follow the ide suggestion